widgetastic_manageiq/__init__.py

```diff
@@ -4073,10 +4073,15 @@ class LineChart(Widget, ClickableMixin):
     tooltip = Table(locator='.//div[contains(@class,"c3-tooltip-container")]/table')
     LEGENDS = ".//*[contains(@class, 'c3-legend-item c3-legend-item-')]"
 
-    def __init__(self, parent, id, logger=None):
+    def __init__(self, parent, id=None, locator=None, logger=None):
         """Create the widget"""
         Widget.__init__(self, parent, logger=logger)
-        self.id = id
+        if id:
+            self.locator = ".//*[@id={}]".format(quote(id))
```

Comment:
OPTIONAL: @digitronik, changes look good but I don't like using such fully abstract xpath.

Reply:
@izapolsk: Accept, but I am not getting supporting elements to improve more static xpath.